openedx/core/djangoapps/content_libraries/api/containers.py

```diff
@@ -437,6 +504,50 @@ def update_container_children(
                         changes=["units"],
                     ),
                 )
+        case ContainerType.Subsection:
+            units = []
+            for key in children_ids:
+                # Verify that all children are units
+                if (
+                    not isinstance(key, LibraryContainerLocator)
+                    or not key.container_type == ContainerType.Unit.value
+                ):
+                    raise ValueError(
+                        f"Invalid children type: {key}. All Subsection children must be Units",
+                    )
+                units.append(_get_container_from_key(key).unit)
+
+            new_version = authoring_api.create_next_subsection_version(
+                container.subsection,
+                units=units,  # type: ignore[arg-type]
+                created=created,
+                created_by=user_id,
+                entities_action=entities_action,
+            )
+
+            # TODO add CONTENT_OBJECT_ASSOCIATIONS_CHANGED for subsections
+        case ContainerType.Section:
+            subsections = []
+            for key in children_ids:
+                # Verify that all children are subsections
+                if (
+                    not isinstance(key, LibraryContainerLocator)
+                    or not key.container_type == ContainerType.Subsection.value
+                ):
+                    raise ValueError(
+                        f"Invalid children type: {key}. All Section children must be Subsections",
+                    )
```

Comment:
Same here - I didn't think this check was necessary.

Reply:
Removed: https://github.com/openedx/edx-platform/pull/36762/commits/02e70f04bad33e95bdcf1c4370b9638184d9d334